content/help/set-up-your-keycard-with-keycard-shell.mdx

```diff
@@ -0,0 +1,113 @@
+---
+id: set-up-your-keycard-with-keycard-shell
+title: Set up your Keycard with Keycard Shell
+---
+
+# Set up your Keycard with Keycard Shell
+
+![Header image showing a Keycard inserted into a Keycard Shell.](/assets/docs/set-up-your-keycard-with-keycard-shell-header.png)
+
+You have two Keycard setup options: [using the Status app](https://status.app/help/keycard/create-a-status-wallet-account-using-a-keycard) ↗, or using Keycard Shell. With Keycard Shell, you can connect your Keycard with other popular [wallet apps](/docs/supported-wallets).
+
+To use your Keycard with Keycard Shell, you need to set a Keycard PIN and optionally a duress PIN, then add a key pair to your Keycard.
+
+<Admonition type="note">
+    The Keycard PIN and duress PIN are two different PINs. The Keycard PIN functions as a password for your Keycard. A duress PIN can unlock a decoy wallet that is indistinguishable from your real wallet. You can use it when someone forces you to unlock your Keycard. Check out [About your Keycard PIN and duress PIN](./about-your-keycard-pin-and-duress-pin) to learn more.
+</Admonition>
+
+Before you start, [get to know your Keycard Shell's controls](./get-to-know-your-keycard-shell).
+
+## Step 1: Set your Keycard PIN and duress PIN
+
+If you do not set a duress PIN during setup, you cannot add one later. This limitation is unavoidable to make the duress PIN undetectable to attackers.
+
+Keycard Shell prevents you from setting your Keycard PIN and duress PIN to be the same.
+
+1. Insert the Keycard into your Keycard Shell with the chip facing up and push it in all the way. Your Keycard Shell powers on automatically.
+1. Set a Keycard PIN and press **OK**.
+1. Enter the Keycard PIN again and press **OK** to confirm.
+1. Optionally, press and hold **OK** to set a duress PIN. Press **C** to skip this step.
+1. Set a duress PIN and press **OK**.
+1. Enter the duress PIN again and press **OK** to confirm.
+1. Press **OK**.
+
+## Step 2: Add a master key pair
+
+There are three methods to add a master key pair to your Keycard:
+- [Generate a new key pair](#method-1-generate-a-new-key-pair)
+- [Import your key pair with a recovery phrase](#method-2-import-your-key-pair-with-a-recovery-phrase)
+- [Import your key pair with a QR code](#method-3-import-your-key-pair-with-a-qr-code)
+
+A key pair consists of a public key (which generates your shareable wallet address) and a private key (which you use to sign transactions and prove ownership). One Keycard can store one master key pair, from which all derived key pairs are created. These derived key pairs corespond to different wallet addresses of your Keycard.
```

Comment:
I think this paragraph works better right after your "Step 2: Add a master key pair" heading.